mumbleBot.py

```diff
@@ -459,6 +516,20 @@ def _download(self, item):
     def loop(self):
         while not self.exit and self.mumble.is_alive():
 
+            # Release started tasks if any
+            if not self.tasks_started:
+                for task in self.task_handle:
+                    self.log.debug(f'bot.loop: about to release task: {task}')
+                    try:
+                        if self.task_handle[task].state[0] == util.TaskState.started:
```

Comment:
I'm kind of confused about "started", "released" and "stopped". Can you explain a little bit?

Reply:
added: init
started: thread is running
released: task may start/resume it's job
paused: task waiting for released
stopped: exit/break task loop

Any renaming suggestions, or would you rather see just threads being released to do their stuff?

As for the interaction with the threads maybe [queue](https://docs.python.org/3.8/library/queue.html) is something to consider here, and feed the q object some addressed TaskHandle , 
For now I will leave it up to further discussion.